### PR TITLE
Only skip julia on OSX

### PR DESCRIPTION
**Pull request by tanglef**
Still not able to run on OSX but all good for linux (improve coverage cf #359)

### CODE REVIEW

File: benchopt/tests/test_benchmarks/dummy_benchmark/test_config.py [hunk 1]
```diff
@@ -12,6 +12,6 @@ def check_test_solver_install(solver_class):
     # conflict with conda packages for R
     # See issue #64
     # julia=1.6.4 broke on conda, see PR #252
-    if 'julia' in solver_class.name.lower():
+    if 'julia' in solver_class.name.lower() and sys.platform == 'darwin':
         # pytest.xfail('Julia causes segfault on OSX for now.')
-        pytest.xfail('Julia install via conda is broken for now.')
+        pytest.xfail('Julia install via conda is broken for now on macos.')
```
Comment (tomMoral) [inline benchopt/tests/test_benchmarks/dummy_benchmark/test_config.py:17]: We should get bck to the commented line
```suggestion=
```

File: benchopt/tests/test_benchmarks/dummy_benchmark/test_config.py [hunk 2]
```diff
@@ -12,6 +12,6 @@ def check_test_solver_install(solver_class):
     # conflict with conda packages for R
     # See issue #64
     # julia=1.6.4 broke on conda, see PR #252
-    if 'julia' in solver_class.name.lower():
+    if 'julia' in solver_class.name.lower() and sys.platform == 'darwin':
         # pytest.xfail('Julia causes segfault on OSX for now.')
```
Comment (tomMoral) [inline benchopt/tests/test_benchmarks/dummy_benchmark/test_config.py:16]:
```suggestion
        pytest.xfail('Julia causes segfault on OSX for now.')
```

File: benchopt/tests/test_benchmarks/dummy_benchmark/test_config.py [hunk 3]
```diff
@@ -12,6 +12,6 @@ def check_test_solver_install(solver_class):
     # conflict with conda packages for R
     # See issue #64
     # julia=1.6.4 broke on conda, see PR #252
```
Comment (tomMoral) [inline benchopt/tests/test_benchmarks/dummy_benchmark/test_config.py:14]: Remove this comment too.
```suggestion
```

**Review by tomMoral — COMMENTED** [review]
A bit a clean up to do but otherwise, looks good

**Review by tomMoral — APPROVED** [review]
LGTM

**Comment by tanglef** [comment]
But it fails now (not before...). Will look into it

**Comment by tomMoral** [comment]
I relaunched the job as it seemed to be an issue with a connection while installing R.

**Pull request merged**